2019-12-31T07:54:54.544Z - error: uncaughtException: Unexpected token ',' date=Tue Dec 31 2019 15:54:54 GMT+0800 (GMT+08:00), pid=8548, uid=null, gid=null, cwd=E:\WebstormProjects\app-backend-test, execPath=C:\Program Files\nodejs\node.exe, version=v12.13.1, argv=[C:\Program Files\nodejs\node.exe, E:\WebstormProjects\app-backend-test\server.js], rss=70520832, heapTotal=58413056, heapUsed=37301120, external=2019045, loadavg=[0, 0, 0], uptime=109547, trace=[column=18, file=internal/modules/cjs/loader.js, function=Module._compile, line=895, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false, column=19, file=internal/modules/cjs/loader.js, function=Module.require, line=852, method=require, native=false, column=18, file=internal/modules/cjs/helpers.js, function=require, line=74, method=null, native=false, column=46, file=E:\WebstormProjects\app-backend-test\server\routes\member.js, function=null, line=4, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false], stack=[E:\WebstormProjects\app-backend-test\server\controller\MemberController.js:110,                 total:total.,,                             ^, , SyntaxError: Unexpected token ',',     at Module._compile (internal/modules/cjs/loader.js:895:18),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14),     at Module.require (internal/modules/cjs/loader.js:852:19),     at require (internal/modules/cjs/helpers.js:74:18),     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\routes\member.js:4:46),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32)]
2019-12-31T08:03:23.300Z - error: GetAllThanksgiving error select * from "member" where "data" = $1 limit $2 - 字段 "data" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=117
2019-12-31T08:03:59.170Z - error: GetAllThanksgiving error select * from "member" where "params" = $1 limit $2 - 字段 "params" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=117
2019-12-31T08:04:37.289Z - error: GetAllThanksgiving error select * from "member" where "sex" = $1 limit $2 - 字段 "sex" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=117
2019-12-31T08:05:38.002Z - error: GetAllThanksgiving error select * from "member" where "sex" = $1 limit $2 - 字段 "sex" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=117
2019-12-31T08:06:27.414Z - error: GetAllThanksgiving error select count(*) from "member" where "query" = $1 - 字段 "query" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=116
2019-12-31T08:06:36.060Z - error: GetAllThanksgiving error select count(*) from "member" where "query" = $1 - 字段 "query" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=116
2019-12-31T08:06:43.953Z - error: GetAllThanksgiving error select count(*) from "member" where "query" = $1 - 字段 "query" 不存在 filePath=E:\WebstormProjects\app-backend-test\server\controller\MemberController.js, lineNumber=116
2019-12-31T09:25:20.710Z - error: uncaughtException: Route.post() requires a callback function but got a [object Object] date=Tue Dec 31 2019 17:25:20 GMT+0800 (GMT+08:00), pid=7240, uid=null, gid=null, cwd=E:\WebstormProjects\app-backend-test, execPath=C:\Program Files\nodejs\node.exe, version=v12.13.1, argv=[C:\Program Files\nodejs\node.exe, E:\WebstormProjects\app-backend-test\server.js], rss=96854016, heapTotal=72331264, heapUsed=51622696, external=2302235, loadavg=[0, 0, 0], uptime=114973, trace=[column=15, file=E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\route.js, function=Route.<computed> [as post], line=202, method=<computed> [as post], native=false, column=19, file=E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js, function=Function.proto.<computed> [as post], line=510, method=<computed> [as post], native=false, column=8, file=E:\WebstormProjects\app-backend-test\server\routes\member.js, function=null, line=21, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false, column=19, file=internal/modules/cjs/loader.js, function=Module.require, line=852, method=require, native=false, column=18, file=internal/modules/cjs/helpers.js, function=require, line=74, method=null, native=false, column=28, file=E:\WebstormProjects\app-backend-test\server\config\route-config.js, function=Object.routeConfig.init, line=6, method=init, native=false, column=17, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=40, method=null, native=false, column=3, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=45, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false], stack=[Error: Route.post() requires a callback function but got a [object Object],     at Route.<computed> [as post] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\route.js:202:15),     at Function.proto.<computed> [as post] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:510:19),     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\routes\member.js:21:8),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14),     at Module.require (internal/modules/cjs/loader.js:852:19),     at require (internal/modules/cjs/helpers.js:74:18),     at Object.routeConfig.init (E:\WebstormProjects\app-backend-test\server\config\route-config.js:6:28),     at E:\WebstormProjects\app-backend-test\server\app.js:40:17,     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\app.js:45:3),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14)]
2019-12-31T09:26:45.261Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:174:3)
    at router (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15 {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:26:59.421Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:174:3)
    at router (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15 {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:27:21.884Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:174:3)
    at router (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15 {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:27:48.381Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:174:3)
    at router (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15 {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:28:04.318Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:174:3)
    at router (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15 {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:41:20.677Z - error: Error: Not Found
    at E:\WebstormProjects\app-backend-test\server\config\error-config.js:10:19
    at Layer.handle [as handle_request] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:317:13)
    at E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:335:12)
    at Immediate.next (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:275:10)
    at Immediate._onImmediate (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:635:15)
    at processImmediate (internal/timers.js:441:21) {
  status: 404
} filePath=E:\WebstormProjects\app-backend-test\server\config\error-config.js, lineNumber=16
2019-12-31T09:45:30.477Z - error: uncaughtException: Route.post() requires a callback function but got a [object Promise] date=Tue Dec 31 2019 17:45:30 GMT+0800 (GMT+08:00), pid=11444, uid=null, gid=null, cwd=E:\WebstormProjects\app-backend-test, execPath=C:\Program Files\nodejs\node.exe, version=v12.13.1, argv=[C:\Program Files\nodejs\node.exe, E:\WebstormProjects\app-backend-test\server.js], rss=98525184, heapTotal=73904128, heapUsed=48329824, external=1888005, loadavg=[0, 0, 0], uptime=116182, trace=[column=15, file=E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\route.js, function=Route.<computed> [as post], line=202, method=<computed> [as post], native=false, column=19, file=E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js, function=Function.proto.<computed> [as post], line=510, method=<computed> [as post], native=false, column=8, file=E:\WebstormProjects\app-backend-test\server\routes\member.js, function=null, line=21, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false, column=19, file=internal/modules/cjs/loader.js, function=Module.require, line=852, method=require, native=false, column=18, file=internal/modules/cjs/helpers.js, function=require, line=74, method=null, native=false, column=28, file=E:\WebstormProjects\app-backend-test\server\config\route-config.js, function=Object.routeConfig.init, line=6, method=init, native=false, column=17, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=40, method=null, native=false, column=3, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=45, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false], stack=[Error: Route.post() requires a callback function but got a [object Promise],     at Route.<computed> [as post] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\route.js:202:15),     at Function.proto.<computed> [as post] (E:\WebstormProjects\app-backend-test\node_modules\express\lib\router\index.js:510:19),     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\routes\member.js:21:8),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14),     at Module.require (internal/modules/cjs/loader.js:852:19),     at require (internal/modules/cjs/helpers.js:74:18),     at Object.routeConfig.init (E:\WebstormProjects\app-backend-test\server\config\route-config.js:6:28),     at E:\WebstormProjects\app-backend-test\server\app.js:40:17,     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\app.js:45:3),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14)]
2019-12-31T09:47:30.975Z - error: uncaughtException: Identifier 'ruleData' has already been declared date=Tue Dec 31 2019 17:47:30 GMT+0800 (GMT+08:00), pid=12208, uid=null, gid=null, cwd=E:\WebstormProjects\app-backend-test, execPath=C:\Program Files\nodejs\node.exe, version=v12.13.1, argv=[C:\Program Files\nodejs\node.exe, E:\WebstormProjects\app-backend-test\server.js], rss=97214464, heapTotal=72331264, heapUsed=51689232, external=2302235, loadavg=[0, 0, 0], uptime=116303, trace=[column=18, file=internal/modules/cjs/loader.js, function=Module._compile, line=895, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false, column=19, file=internal/modules/cjs/loader.js, function=Module.require, line=852, method=require, native=false, column=18, file=internal/modules/cjs/helpers.js, function=require, line=74, method=null, native=false, column=17, file=E:\WebstormProjects\app-backend-test\server\routes\member.js, function=null, line=7, method=null, native=false, column=30, file=internal/modules/cjs/loader.js, function=Module._compile, line=959, method=_compile, native=false, column=10, file=internal/modules/cjs/loader.js, function=Module._extensions..js, line=995, method=.js, native=false, column=32, file=internal/modules/cjs/loader.js, function=Module.load, line=815, method=load, native=false, column=14, file=internal/modules/cjs/loader.js, function=Module._load, line=727, method=_load, native=false, column=19, file=internal/modules/cjs/loader.js, function=Module.require, line=852, method=require, native=false, column=18, file=internal/modules/cjs/helpers.js, function=require, line=74, method=null, native=false, column=28, file=E:\WebstormProjects\app-backend-test\server\config\route-config.js, function=Object.routeConfig.init, line=6, method=init, native=false, column=17, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=40, method=null, native=false, column=3, file=E:\WebstormProjects\app-backend-test\server\app.js, function=null, line=45, method=null, native=false], stack=[E:\WebstormProjects\app-backend-test\server\middleware\rules.js:6,     let ruleData = [,         ^, , SyntaxError: Identifier 'ruleData' has already been declared,     at Module._compile (internal/modules/cjs/loader.js:895:18),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14),     at Module.require (internal/modules/cjs/loader.js:852:19),     at require (internal/modules/cjs/helpers.js:74:18),     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\routes\member.js:7:17),     at Module._compile (internal/modules/cjs/loader.js:959:30),     at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10),     at Module.load (internal/modules/cjs/loader.js:815:32),     at Function.Module._load (internal/modules/cjs/loader.js:727:14),     at Module.require (internal/modules/cjs/loader.js:852:19),     at require (internal/modules/cjs/helpers.js:74:18),     at Object.routeConfig.init (E:\WebstormProjects\app-backend-test\server\config\route-config.js:6:28),     at E:\WebstormProjects\app-backend-test\server\app.js:40:17,     at Object.<anonymous> (E:\WebstormProjects\app-backend-test\server\app.js:45:3)]
